src/kid.js

```diff
@@ -39,6 +55,12 @@ function checkModulo(modulo, { mod10, mod11 }, ...opts) {
   return finalMod
 }
 
+/**
+ * 
+ * @param {String} kid The kid number
+ * @param {String} modulo The modulo 
```

Comment:
`{String} modulo The modulo` is ambigous. See other comment

Reply:
fixed by f9aa173